api/metric/aggregation/descriptor.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aggregation
+
+//go:generate stringer -type=Operator
+type Operator int
+
+const (
+	NONE Operator = iota
+	SUM
+	COUNT
+	MIN
+	MAX
+	LAST_VALUE
+	DISTRIBUTION
```

Comment:
For a distribution if a histogram is used user requires to also configure the buckets. If a summary is used user requires to configure the percentiles. So I cannot see how this is used.